src/Components/LifecycleTable/LifecycleTable.tsx

```diff
@@ -283,18 +324,47 @@ export const LifecycleTable: React.FunctionComponent<LifecycleTableProps> = ({
           {viewFilter !== 'all' && (
             <Td dataLabel={SYSTEM_LIFECYCLE_COLUMN_NAMES.count}>
               {repo.count !== 0 ? (
-                <Button
-                  variant="link"
-                  onClick={(event) => {
-                    handleModalToggle(event);
-                    setModalDataName(String(repo.name));
-                    setModalData(repo.systems);
-                  }}
-                >
-                  {repo.count}
-                </Button>
+                <div style={{ display: 'flex', alignItems: 'center', gap: '4px' }}>
+                  <StatusIcon supportStatus={repo.support_status || ''} />
+                  <Button
+                    variant="link"
+                    style={{
+                      margin: 0,
+                      padding: 0,
+                      paddingLeft:
+                        repo.support_status === 'Supported' ||
+                        repo.support_status === 'Support ends within 6 months' ||
+                        repo.support_status === 'Retired'
+                          ? '0'
+                          : '18px',
+                    }}
+                    onClick={(event) => {
+                      handleModalToggle(event);
+                      setModalDataName(String(repo.name));
+                      setModalData(repo.systems);
+                    }}
+                  >
+                    {repo.count}
+                  </Button>
+                </div>
               ) : (
-                <>{repo.count}</>
+                <div style={{ display: 'flex', alignItems: 'center', gap: '4px' }}>
+                  <StatusIcon supportStatus={repo.support_status || ''} />
+                  <span
+                    style={{
+                      margin: 0,
+                      padding: 0,
+                      paddingLeft:
+                        repo.support_status === 'Supported' ||
+                        repo.support_status === 'Support ends within 6 months' ||
+                        repo.support_status === 'Retired'
+                          ? '0'
+                          : '18px',
+                    }}
+                  >
+                    {repo.count}
+                  </span>
+                </div>
```

Comment:
```suggestion
              <> {repo.count}</>  
```
this seems like false condition of the ternary operator `repo.count !== 0` I would say we should have just the number without possibility to open modal. It might be a possibility in "installed&related". Could you please add the `repo.count !== 0 ? <button></button> : {repo.count}` (I simplified that) check also to the app streams table?